protobufs/config/service.proto

```diff
@@ -32,7 +32,7 @@ service ConfigService {
   }
   rpc GetAllTenants(google.protobuf.Empty) returns (GetAllTenantResponse) {
     option (google.api.http) = {
-      get: "/config/tenant/list"
+      get: "/config/tenant"
```

Comment:
bugfix

Reply:
IMO we need a plural verb here, either what it was or `/config/tenants`

Reply:
agree, it needs to change for all endpoints per rest conventions.